RELEASE.md

```diff
@@ -16,6 +16,7 @@
 * You can configure config file patterns through `settings.py` without creating a custom config loader.
 
 ## Bug fixes and other changes
+* Added `VideoDataSet` to read and write video files and interact with their frames
```

Comment:
I think we should move this to the `Major features and improvements` section above and format it in a table similar to the added datasets here https://github.com/kedro-org/kedro/blob/main/RELEASE.md#major-features-and-improvements-7

Reply:
done.